apps/condo/domains/settings/schema/MobileFeatureConfig.js

```diff
@@ -0,0 +1,111 @@
+
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/settings/access/MobileFeatureConfig')
+
+
+
+const ERRORS = {
+    TICKET_SUBMITTING_PHONES_NOT_CONFIGURED: {
+        code: BAD_USER_INPUT,
+        type: 'TICKET_SUBMITTING_PHONES_NOT_CONFIGURED',
+        message: 'commonPhone field not specified',
+        messageForUser: 'api.organization.MobileFeatureConfig.TICKET_SUBMITTING_PHONES_NOT_CONFIGURED',
+    },
+    COMMON_PHONE_INVALID: {
+        code: BAD_USER_INPUT,
+        variable: ['data', 'commonPhone'],
+        type: 'COMMON_PHONE_NOT_CONFIGURED',
+        message: 'commonPhone is invalid',
+        messageForUser: 'api.organization.MobileFeatureConfig.COMMON_PHONE_INVALID',
+    },
+    ORGANIZATION_IS_REQUIRED: {
+        code: BAD_USER_INPUT,
+        variable: ['data', 'organization'],
+        type: 'ORGANIZATION_IS_REQUIRED',
+        message: 'The organization is required to create',
+        messageForUser: 'api.organization.MobileFeatureConfig.ORGANIZATION_IS_REQUIRED',
+    },
+}
+
+const MobileFeatureConfig = new GQLListSchema('MobileFeatureConfig', {
+    schemaDoc: 'The entity helps to manage the enabling/disabling of some features in the mobile application,' +
+        ' as well as to store the data required in the disabled state of the feature.',
+    fields: {
+
+        organization: {
+            schemaDoc: 'Ref to the organization. Also needed for scope management',
+            type: Relationship,
+            ref: 'Organization',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        commonPhone: {
+            schemaDoc: 'The phone number where the organization wants to receive common calls',
+            type: Text,
+        },
+
+        ticketSubmittingIsDisabled: {
+            schemaDoc: 'Disabling the ability to send tickets by the user of the resident\'s mobile application. ' +
+                'Instead, he will be shown a screen with phones where he can contact to send a request',
+            type: Checkbox,
+            hooks: {
+                validateInput: async ({ resolvedData, context }) => {
+                    if (resolvedData.ticketSubmittingIsDisabled) {
+                        if (!resolvedData['commonPhone']) {
+                            throw new GQLError(ERRORS.TICKET_SUBMITTING_PHONES_NOT_CONFIGURED, context)
+                        }
+                        if (resolvedData['commonPhone'] && normalizePhone(resolvedData['commonPhone']) !== resolvedData['commonPhone']) {
+                            throw new GQLError(ERRORS.COMMON_PHONE_INVALID, context)
+                        }
+                    }
+                },
+            },
+        },
+
+        onlyProgressionMeterReadingsIsEnabled: {
```

Comment:
"…IsEnabled" is a redundant suffix, because the field is a Checkbox.

```suggestion
        onlyProgressionMeterReadings: {
```